api/app/tests/apps/signals/test_api_v1_endpoints.py

```diff
@@ -422,6 +506,7 @@ def test_create_initial_and_upload_image(self):
         response = self.client.post(new_image_url, data={'image': image})
         self.assertEqual(response.status_code, 403)
 
+    @skip('Updates not yet supported')
```

Comment:
Remove this skip, make sure the location update flow also implements the address check.

- should be possible to post coordinates and addres
- should be possible to post only coordinates

- should not be possible to post only address